### DISCUSSION TITLE
Calculate correlation from adonis using Bray-Curtis

### DISCUSSION THREAD

**Discussion by pauGuas** [vegandevs/vegan#490] [Q&A]
I am analyzing phyloseq data to figure out what influences microbial communities. I have significant results for "season" and "basin" with adonis.  Now, I need to find out how much those variables correlate with the community composition. Can anyone help me with that?

**Comment by jarioksa** [comment]
The output has column `R2` which is the squared correlation correlation coefficient. For whatever it's worth.
